src/pages/Dashboard/container.js

```diff
@@ -9,6 +9,10 @@ import eventsView from '../../utilities/eventsView';
 import moment from 'moment';
 import eventTypes from '../../utilities/eventTypes';
 import holidayStatus from '../../utilities/holidayStatus';
+import {checkIfPastDatesSelected,
+  checkIfDatesFallOnWeekend,
```

Comment:
It seems to be that Utilities is concerned with a number of different things. In this class alone I can see it supplying:
1. constants
2. extensions for toast
3. business logic

Not for this PR, but I think some of these things would be within services (business logic), some within constants (event types and holiday status) and the rest staying in utilities.

Reply:
Yea I was discussing this with Kirsty last week. There's definitely some tech debt there to split things out.